<launch>
    <node name="node_data_for_vision_perception" pkg="draconis_demo_datafeed" type="node_data_for_vision_perception.py" output="screen">

        <param name="topic_input_camera_front" value="/$(env ROBOT_NAMESPACE)/argus/ar0234_front_right/image_raw" />
        <param name="topic_input_camera_left" value="/$(env ROBOT_NAMESPACE)/argus/ar0234_side_left/image_raw" />
        <param name="topic_input_camera_right" value="/$(env ROBOT_NAMESPACE)/argus/ar0234_side_right/image_raw" />
        <param name="topic_input_camera_rear" value="/$(env ROBOT_NAMESPACE)/argus/ar0234_rear/image_raw" />
        <param name="topic_input_velodyne" value="/$(env ROBOT_NAMESPACE)/velodyne_points" />

        <param name="topic_output_images_and_cloud" value="/$(env ROBOT_NAMESPACE)/images_and_cloud" />
        
        <param name="output_framerate" value="4" type="double"/>
        <param name="images_and_cloud_package_frame_id" value="$(env ROBOT_NAMESPACE)/base_link" type="str" />
    </node>
</launch>